### ISSUE TITLE
FlutterSemanticsScrollView exposing unlabeled unnecessary accessibility node on iOS

### ISSUE THREAD

**Issue by henriquegrandinetti** [flutter/flutter#99011]
## Steps to Reproduce

1. Execute a flutter application with a scrollable view
2. Turn on iOS accessibility inspector (emulator) or VoiceOver (device)
3. Navigate through the semantic nodes

**Expected results:**
No node is created for the scrollable view (the wrapper view, not the items it contains).

**Actual results:**
An accessibility node is created, without any label. This cal also by reproducible by having an EarlGrey tests that check accessibility elements.

Additional details on go/fluttersemanticsbug.

**Comment by henriquegrandinetti** [comment]
Yes, you can run the tests locally to reproduce. I added more details on go/fluttersemanticsbug.

**Comment by doc-rj-ebay** [comment]
@henriquegrandinetti Thanks for reporting this. I'm currently dealing with it. Is there a workaround? I have a SingleChildScrollView and VoiceOver says "Description for element unavailable" for the unwanted accessibility node.

**Comment by chriscarrillo** [comment]
I'm experiencing this issue.

@chunhtai Do you have an idea of when this issue will be addressed?

**Comment by akaegi** [comment]
Same issue for me...

**Comment by AhmadFayyas** [comment]
My observation is that this issue is reproducible in Flutter’s Accessibility Inspector, making it seem like an accessibility bug. However, **real devices using screen readers (TalkBack, VoiceOver) do not exhibit the same behavior.**

Flutter’s Accessibility Inspector is not 100% accurate in reflecting real-world screen reader behavior. Preferably, test accessibility features on real devices (TalkBack for Android, VoiceOver for iOS) and verify that they work as expected!

**Comment by mahmoudelbanna** [comment]
Hi,

When using CustomScrollView or any scrollable widget on iOS, we’ve observed that a FlutterSemanticsScrollView node is automatically added by the platform embedding.

This semantic node creates a container box before the first scroll item.
If the user taps a non-interactive (empty) area while VoiceOver is on, the semantic focus jumps to this scroll container.
This causes the semantic order to shift, making it difficult for users to navigate back to previously visible content (especially items that have been scrolled out of view).
This issue only occurs on iOS — it does not happen on Android with TalkBack.
Is there any way to disable or override the insertion of this FlutterSemanticsScrollView node on iOS?
Or at least prevent it from receiving focus when tapping on empty space?

This behaviour is negatively impacting accessibility for VoiceOver users, and we’d greatly appreciate any suggestions, workarounds, or guidance.

Tested on:

- iPhone 13 (iOS 18.5)

-  Flutter version 3.32.4

Thanks in advance!

**Comment by agustinTP** [comment]
This is still happening on ios, we use flutter for an education app that requires by law to have adequate semantics and screen reader options and we would be really grateful on finding a solution for this unwanted node.

Tested it on simulator with accessibility inspector and keeps happening
Also tested deploying on physical device (iphone 16 pro) and also happens

**Comment by akaegi** [comment]
Maybe this is a duplicate of https://github.com/flutter/flutter/issues/164483? The latter seems to have been fixed recently :-)

**Comment by MuneerAhamed01** [comment]
Guys Any update on this :)

**Comment by hannah-hyj** [comment]
mark this as a duplicate of https://github.com/flutter/flutter/issues/164483
